src/types/onyx/Card.ts

```diff
@@ -8,18 +8,22 @@ type Card = {
     bank: string;
     availableSpend: number;
     domainName: string;
-    maskedPan?: string; // do not reference, removing as part of Expensify/App#27943
     lastFourPAN?: string;
-    cardName: string;
-    cardTitle: string; // used only for virtual limit cards
-    limitType: ValueOf<typeof CONST.EXPENSIFY_CARD.LIMIT_TYPES>;
-    isAdminIssuedVirtualCard: boolean;
     isVirtual: boolean;
     fraud: ValueOf<typeof CONST.EXPENSIFY_CARD.FRAUD_TYPES>;
-    cardholderFirstName: string;
-    cardholderLastName: string;
     errors?: OnyxCommon.Errors;
     isLoading?: boolean;
+    nameValuePairs?: {
+        limitType?: ValueOf<typeof CONST.EXPENSIFY_CARD.LIMIT_TYPES>;
+        cardTitle?: string; // used only for virtual limit cards
```

Comment:
Curious if we're calling `admin-issued virtual cards` as `virtual limit cards` in-product?

Reply:
Nope, I got mixed up. I'll fix that!